roles/storage-cinder/tasks/deprovision.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Delete Service Account
+  command: kubectl delete serviceaccount {{service_account}} -n {{namespace}}
+  when: sa.stdout == ""
+
+- name: Delete {{privileged_service_account}} serviceaccount
+  command: kubectl create serviceaccount {{privileged_service_account}} -n {{ namespace }}
+  when: psa.stdout == ""
+
+- name: Select a host for Mariadb hostpath
+  command: kubectl get nodes --selector="{{infra_node_label}}"=true
+  register: infra_node_hostnames
+
+- name: Remove label on the nodes that host Mariadb
+  command: kubectl label node "{{ item }}" "{{infra_node_label}}"-
+  with_items: "{{ infra_node_hostnames.stdout.split() }}"
+
```

Comment:
will Delete and Remove tasks pass if there is no account/label? no --ignore-errors or similar flag needed?

Reply:
The command to remove the label will still work even though the label is missing. It will print out a message saying that the label isn't present, but the return code is still 0.